sdk/search/Microsoft.Azure.Search/changelog.md

```diff
@@ -0,0 +1,227 @@
+# GA SDK Releases
+
+Every minor version release targets the same (GA) API version as that of the corresponding major version release. API version targeted by the major version releases are listed when appropriate.
+
+Features and improvements in a GA SDK are considered generally available, unless otherwise specified.
+
+<a name="10.0.0"></a>
+
+## 10.0.0

```

Comment:
@scbedd can confirm but I believe the version number needs to be under a heading 1 section (i.e. only 1 `#`) to be properly detected.